src/microbe_stage/MicrobeStage.cs

```diff
@@ -44,6 +44,12 @@ public class MicrobeStage : StageBase<Microbe>
     [JsonProperty]
     private float elapsedSinceEntityPositionCheck;
 
+    /// <summary>
+    ///   Used to control how often light level is updated by the day/night cycle.
+    /// </summary>
+    [JsonProperty]
```

Comment:
Why not JsonIgnore?

Plus I'd prefer it to be named as timeElapsedSinceLastLightLevelUpdate

Reply:
Both of these match `elapsedSinceEntityPositionCheck` above, so that should be changed too if this is the decision.

Reply:
Fine then.